tests/test_primerdesign.py

```diff
@@ -530,6 +531,7 @@ def test_misprime_lib_mishyb_lib(self):
         )
         self.assertEqual(result['PRIMER_PAIR_NUM_RETURNED'], 5)
         self.assertEqual(result['PRIMER_LEFT_0'], [46, 21])
+        self.assertEqual(result['PRIMER_LEFT'][0]['COORDS'], [46, 21])
```

Comment:
Is there a reasonably simple way to extend this test to ensure complete parity between the nested entries and the non-nested entries? I guess there must not be, otherwise the changes to `thermoanalysis` would have been simpler / not inline (i.e., just a loop / dict comprehension)

Reply:
Yeah it feels like we need to somehow assert parity between the two (maybe there is some sort of static LUT that makes field mappings explicit in a documentation-as-code type of situation?).